core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -1975,9 +1976,9 @@ class SparkContext(config: SparkConf) extends Logging {
     } else {
       val (keys, scheme) = if (path.contains("\\") && Utils.isWindows) {
         // For local paths with backslashes on Windows, URI throws an exception
-        (addLocalJarFile(new File(path)), "local")
+        (addLocalJarFile(convertPathToFileForWin(path)), "local")
```

Comment:
@sarutak, I think here we assume the path is always a local path on Windows (not an URI) so I guess it's fine to leave it (?)

Reply:
Ah, seems so. if the path contains "\\", it's Windows' local path.
I'll leave it.